pages/linux/pacman-deptest.md

```diff
@@ -0,0 +1,20 @@
+# pacman --deptest
+
+> Check each dependency specified and return a list of dependencies that are not currently satisfied on the system..
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Display help:
+
+`pacman --deptest --help`
+
+- Out of a list of packages, print the names of those that aren't installed:
```

Comment:
```suggestion
- Output of a list of packages, print the names of those that aren't installed:
```

Reply:
This verb is not good enough for a non-native speaker like me, could we use `Display`? `output` should be a noun not a verb :smile: 
```suggestion
- Display a list of packages, print the names of those that aren't installed:
```

Reply:
Totally! Go right ahead. 

Yes :smile: that's just one of the horrible breaking of it's own rules that English has 🤷🏻